docs/api/non_geo_datasets.csv

```diff
@@ -0,0 +1,33 @@
+Dataset,Task,Source,# Samples,# Classes,Size (px),Resolution (m),Bands
+`ADVANCE`_,C,"Google Earth, Freesound",5075,13,512x512,0.5,RGB
+`Benin Cashew Plantations`_,S,Airbus Pléiades,70,6,"1,186x1,122",0.5,MSI
+`BigEarthNet`_,C,Sentinel-1/2,590326,19--43,120x120,10,"SAR, MSI"
+`COWC`_,"C, R","CSUAV AFRL, ISPRS, LINZ, AGRC",388435,2,256x256,0.15,RGB
+`Kenya Crop Type`_,S,Sentinel-2,4688,7,"3,035x2,016",10,MSI
+`DFC2022`_,S,Aerial,,15,"2,000x2,000",0.5,RGB
+`ETCI2021 Flood Detection`_,S,Sentinel-1,66810,2,256x256,5-20,SAR
+`EuroSAT`_,C,Sentinel-2,27000,10,64x64,10,MSI
+`FAIR1M`_,OD,Gaofen/Google Earth,15000,37,"1,024x1,024",0.3-0.8,RGB
+`Forest Damage`,OD,Drone imagery,1543,4,1500x1500,,RGB
+`GID-15`_,S,Gaofen-2,150,15,"6,800x7,200",3,RGB
+`IDTReeS`_,"OD,C",Aerial,591,33,200x200,0.1-1,RGB
+`Inria Aerial Image Labeling`_,O,Aerial,360,,5000x5000,0.3,RGB
+`LandCover.ai`_,S,Aerial,10674,5,512x512,0.25-0.5,RGB
+`LEVIR-CD+`_,CD,Google Earth,985,2,"1,024x1,024",0.5,RGB
+`LoveDA`_,S,Google Earth,5987,7,"1,024x1,024",0.3,RGB
+`NASA Marine Debris`_,OD,PlanetScope,707,1,256x256,3,RGB
+`NWPU VHR-10`_,I,"Google Earth, Vaihingen",800,10,"358--1,728",0.08-2,RGB
+`OSCD`_,CD,Sentinel-2,24,2,"40--1,180",60,MSI
+`PatternNet`_,C,Google Earth,30400,38,256x256,0.06-5,RGB
+`Potsdam`_,S,Aerial,38,6,"6,000x6,000",0.05,MSI
+`RESISC45`_,C,Google Earth,31500,45,256x256,0.2-30,RGB
+`Seasonal Contrast`_,T,Sentinel-2,100K--1M,,264x264,10,MSI
+`SEN12MS`_,S,"Sentinel-1/2, MODIS",180662,33,256x256,10,"SAR, MSI"
+`So2Sat`_,C,Sentinel-1/2,400673,17,32x32,10,"SAR, MSI"
+`SpaceNet`_,I,WorldView-2/3 Planet Lab Dove,"1,889--28,728",2,102--900,0.5-4,MSI
+`Tropical Cyclone`_,R,GOES 8--16,108110,,256x256,4K—8K,MSI
```

Comment:
```suggestion
`Tropical Cyclone`_,R,GOES 8--16,108110,,256x256,4K--8K,MSI
```
I think this was a em-dash unicode char, should be en-dash

Reply:
Thanks, thought I got all these back to hyphens